proto/squad/liquidfarming/v1beta1/tx.proto

```diff
@@ -100,3 +104,14 @@ message MsgRefundBid {
 
 // MsgRefundBidResponse defines the MsgRefundBidResponse response type.
 message MsgRefundBidResponse {}
+
+// MsgAdvanceAuction defines a message to advance rewards auction by one.
```

Comment:
Is this only for testing?

Reply:
Yes. `MsgAdvanceAuction` is only available when you build with `$ make install-testing`